.env

```diff
@@ -0,0 +1,3 @@
+DTSCATALOG=tests/catalog/example-collection.xml
+SERVER_HOST=0.0.0.0
+SERVER_PORT=5000
```

Comment:
note: on MacOS port 5000 is used by AirPlay Receiver, please consider using an alternative port (ex: 8000)